src/EA.Weee.Web/Areas/AATFReturn/Views/ReusedRemoveSite/Index.cshtml

```diff
@@ -32,24 +33,32 @@
                         Site address
                     </div>
                     <div class="govuk-grid-column-one-half weee-tonnage-column-padding-left" style="font-weight: normal">
+                        @Html.HiddenFor(m => m.SiteAddress)
                         @Html.Raw(Model.SiteAddress.Replace("\r\n", "<br />"))
                     </div>
                 </div>
             </div>
         </div>
-
-        <div class="govuk-grid-column-full govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" style="padding-left: 0px; margin-bottom: 10px;">
-            @this.WeeeGds().RadioButtonsFor(
-                m => m.SelectedValue,
-                Model.PossibleValues,
-                "Confirm if you want to remove this site from your list",
-                RadioButtonLegend.VisuallyHidden,
-                RadioButtonLayout.Inline)
+        <div class="govuk-form-group" style="padding-bottom: 10px">
+            <div class="govuk-grid-column-full govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" style="padding-left: 0px; margin-bottom: 10px;">
+                @this.WeeeGds().RadioButtonsFor(
+                    m => m.SelectedValue,
+                    Model.PossibleValues,
+                    "Confirm if you want to remove this site from your list",
+                    RadioButtonLegend.VisuallyHidden,
+                    RadioButtonLayout.Inline)
+            </div>
+            <img src="~/Content/govuk_frontend/images/icon-important.png" alt="icon-important" style="float:left;" height="42" width="42">
+            <p class="important">You may need to update the reuse tonnage values you have entered</p>
         </div>
-        <p class="govuk-body-s">You may need to update the reuse tonnage values you have entered</p>
         <button type="submit" name="save" class="govuk-button" id="site-details">
             Save and continue
         </button>
-    }
+        }
 
-</section>
\ No newline at end of file
+</section>
```

Comment:
see my comment [here](https://github.com/DEFRA/prsd-weee/pull/219#discussion_r283250340) about moving this to an scss file